### DISCUSSION TITLE
From the dashboard there is no direct access my repo list, I need to type my name in the URL

### DISCUSSION THREAD

**Discussion by ppazos** [community/community#122227] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Add a link to access the user's repo list from the dashboard. Seems pretty basic.

**Comment by davevad93** [comment]
Hi @ppazos ,

I don't know if this is what you're looking for, but there's a way to access directely your repos from the dashboard, on the left side if you focus on the search bar, it will display a list of your repositories and repos you have access or have contributed to:

![000](https://github.com/community/community/assets/94308234/6db12e97-b07d-45e3-80d7-3f5b643cd378)

**Reply by davevad93** [reply]
Ok, I realized that maybe you already knew it and you meant just that, so my answer was redundant, I apologize for not understanding your question properly.

**Reply by ppazos** [reply]
@davevad93 yes there's a search, my comment is about there is no access to the repositories list: the full list.

**Reply by ppazos** [reply]
@davevad93 I mean to add a link to this page https://github.com/ppazos?tab=repositories in the dashboard.

**Reply by davevad93** [reply]
Hey @ppazos ,

Ok, now I understand what you're saying and yes, it would be a pretty good feature to add to the dashboard. It's quite odd that it hasn't already been implemented, because as you  said it's something basic.

**Reply by ppazos** [reply]
I agree @davevad93 I think a good place is at the bottom of the image you shared, where there is a summary of the repos, to add an "All repositories" button or link that goes to the repo list would be useful. In my case, some times I forgot the names of the repos and need to check the description too, or have several repos with similar names, and again I need to check other details besides the name. So the summary list with the search is great when you know the exact name of the repo, and it's quicker, but when you get to a certain age you start forgetting things hehe...

**Reply by ebndev** [reply]
Hi @ppazos (& @rofrano),

When you click your profile photo on the top right on the page, there is a link to "Your Repositories" that's the full list of repos you shared in the link above. Is this the action you wanted to do?

Please clarify if not!

<img width="320" alt="Screenshot 2024-05-09 at 4 52 17 PM" src="https://github.com/community/community/assets/164093511/dc6383f9-1ee8-444e-9484-aa796db58f11">

**Reply by ppazos** [reply]
Yes that's the action. Kind of hidden IMO for something I use all the time, more than the summary on the left with the search, which I don't use much and takes a lot of screen space. I would prefer a button shown on the screen to "my repositories" in the dashboard.

**Reply by rofrano** [reply]
@ebndev Yes! Nicely hidden. Now I'm down to 2 clicks. I'm with @ppazos , Maybe we can get this on the Dashboard so it's only 1 click? 

I would gladly give up "the feed" to be able to get to my code faster because I come to GitHub to code, not to scroll through a feed of what everyone else is doing that I'm not interested in.

Something to consider: What is the purpose of GitHub? It is to work on your code. What is the largest thing on the Dashboard? A social media feed! Can you see the problem? It consumes the entire page! This is not Twitter to Facebook. I don't come to GitHub to keep up on what people are doing. I come to GitHub to code. You would think that the largest real estate on the dashboard would be dedicated to finding the code I need to work on... but it's not. Just something for the GitHub UX people to consider. 

How about a dashboard where I can arrange the organizations that I belong to in an order that makes sense to me, and then I can easily get to the repos in that org easily. That would be an outstanding dashboard for a software engineer looking to get to their code quickly. Maybe have a section on the side that contains the status of their GitHub Actions so they can quickly see if builds are broken? Or a security section to see what vulnerabilities need to be fixed in my repos? Those are things I'm interested in on a dashboard. Just say'in. 

Thanks for the tip. I do appreciate only having to do 2 clicks to get to my repos.

**Reply by ppazos** [reply]
That's a good one, I don't read the feed, for me it's just noise . And totally agree on "getting to the code fastest"!

**Comment by rofrano** [comment]
Hi @davevad93, Add my vote for that feature because. actually... I came here to suggest exactly the same thing! There is NO WAY to just see my repos from the Dashboard. The "Top Repositories" are repositories that I have access to. Not just MY repositories but all of the repos in all of the organizations that I have access to, and many times my personal account repos are not even listed at the top.

### The Problem

It takes **4 clicks** to see my personal repos every time I come to GitHub.

This is what I do every time I come to GitHub and want to see my repos:

1. Remember the name of a repository that is associated with my personal account and type it into Top Repositories
2. Then Select that repo
3. Then select my Account name from the top of that repo page (this gets me to my account page)
4. Finally click on the Repositories tab on my Account page to list my repos

That's a lot of clicks. I feel that listing my repos should be on the Dashboard. Personally, I completely ignore "the feed". It just gets in the way. I wish there was a way to turn it off. If I could replace the feed with a list of repos that would be extremely valuable. 

So, I have to agree 100% with @ppazos ...**From the dashboard there is no direct access my repo list!** (which is a pretty basic feature)

**Reply by LiteBrite82** [reply]
@rofrano 
This is excellent feedback worthy of its own discussion. I recommend starting a new discussion with a focused title so it is more visible to the community and easier to upvote.

**Reply by LiteBrite82** [reply]
> I would gladly give up "the feed" to be able to get to my code faster because I come to GitHub to code, not to scroll through a feed of what everyone else is doing that I'm not interested in.

> Something to consider: What is the purpose of GitHub? It is to work on your code. What is the largest thing on the Dashboard? A social media feed! Can you see the problem? It consumes the entire page! This is not Twitter to Facebook. I don't come to GitHub to keep up on what people are doing. I come to GitHub to code. You would think that the largest real estate on the dashboard would be dedicated to finding the code I need to work on... but it's not. Just something for the GitHub UX people to consider.

> How about a dashboard where I can arrange the organizations that I belong to in an order that makes sense to me, and then I can easily get to the repos in that org easily. That would be an outstanding dashboard for a software engineer looking to get to their code quickly. Maybe have a section on the side that contains the status of their GitHub Actions so they can quickly see if builds are broken? Or a security section to see what vulnerabilities need to be fixed in my repos? Those are things I'm interested in on a dashboard. Just say'in.

**Reply by ejenk0** [reply]
I could not agree more. This is something that bugs me every time I visit Github. My personal workflow is usually:
1. I want to visit one of my many personal repositories
2. Type "github" into my browser which brings me to my dashboard
3. I have disabled most of the "Feed", so almost all of my screen space is totally wasted. The repositories listed on the left seem to be in no particularly useful order, and are incomplete.
4. I click my profile icon in the top right, then "Your Repositories"
5. This view is decent, and usually gets me where I want

I wish that I could configure my dashboard to simply show me a compact grid of my recently changed repositories. This would make my GitHub usage much smoother for me.

**Reply by rofrano** [reply]
> This is excellent feedback worthy of its own discussion.

@LiteBrite82 I'd be happy to start a new discussion. What topic should I choose? I don't see a Dashboard topic! This one is Feed but I'm really talking about creating a more useful Dashboard (i.e., including those areas outside the feed) What's a good topic to post under? (and why isn't there a Dashboard topic?)

**Reply by LiteBrite82** [reply]
@rofrano 
I think the `General` category would be the most beneficial for the discussion visibility if it is focused on the UI/UX of the dashboard as a whole. 

In trying to keep this current thread on topic, I'm tempted to hide this new development we have been discussing (as off topic) simply for separation of concerns ( Once the new discussion is started). I dont want to detract from @ppazos original discussion, and we track community interest in a discussion by upvotes. I think both of these discussions have a lot of value and I want to make sure they are tracked accurately for the community. 

Since there is a bit of correlation, maybe we leave a link to the new discussion in this thread once it is created?

**Reply by LiteBrite82** [reply]
I also just noticed that this post is in `Feed` and should be in `General`. Moving it now. ![hold on](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExd2s5ZW54N2FnZ3lyMmJmbzBvYXd2bzF5emswbWcyMTFiN2d1dHRwbCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/OCu7zWojqFA1W/giphy.gif)
